src/index.js

```diff
@@ -0,0 +1,88 @@
+/** @format */
+
+import './style.css';
+import store from './modules/storage.js';
+import {
+  addTask, displayTask, clearField, input,
+} from './modules/utility.js';
+
+const addBtn = document.querySelector('#addBtn');
+const clearBtn = document.querySelector('#clearBtn');
+
+// Display
+window.addEventListener('load', () => {
+  displayTask(store.tasks);
+});
+
+// add
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  if (input.value === '') {
+    // console.log("All Fields are required");
+  } else {
```

Comment:
The code should be clean and shouldn't have console.log. The comment command is not used properly, and it makes your code look untidy.